File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/model/Name.java
package com.tistory.black_jin0427.myandroidarchitecture.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Name implements Serializable {

    @SerializedName("title") public String title;

    @SerializedName("first") public String first;

    @SerializedName("last") public String last;
}

File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/model/Name.kt
package com.tistory.black_jin0427.myandroidarchitecture.api.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Name  (

    @SerializedName("title")
    var title: String?,

    @SerializedName("first")
    var first: String?,

    @SerializedName("last")
    var last: String?

): Serializable
